app/models/saved_claim/dependency_claim.rb

```diff
@@ -184,6 +180,10 @@ def upload_to_vbms(path:, doc_type: nil)
     )
 
     uploader.upload! unless Rails.env.development?
+  rescue
```

Comment:
If this is not enough to track down the source of the increase in VBMS errors we're seeing, we might need to either reenable the error logging (with a flipper, but trusting to the PII scrubber for protection), or find a way to store the VBMS error securely/encrypted for review.